salesagility/SuiteCRM
# #9820 -Update EmailTemplate.php

Allows currency fields  to be parsed in email templates as currency on the final email.



The current $value just outputs raw value. Needs to be formatted as currency. There is already a function in the file to do that, just need to get the field type to use the function.

## Description




First lirst line just gets the field "type" from the array. The second field just runs the value through the already defined function _convertToType() which formats for currency if it is a currency field.

This is issue https://github.com/salesagility/SuiteCRM/issues/9820ine just gets the field "type" from the array. The second field just runs the value through the already defined function _convertToType() which formats for currency if it is a currency field.

This is issue https://github.com/salesagility/SuiteCRM/issues/9820
## Motivation and Context

It solves the problem of email templates with currency being formatted properly.
## How To Test This

I tested this with a currency field and also a text field and both work after the change and produce the desired result in the email template final formatting.

To test you can create an email template with a currency field and see it is not formatted as currency, apply the patch and then re-output the email and you'll see its formatted as currency.

## Types of changes

- [ x] Bug fix (non-breaking change which fixes an issue)

### Final checklist


- [ x] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ x] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.
